packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -477,6 +537,77 @@ contract L1StandardBridge_DepositETH_Test is L1StandardBridge_TestInit {
         vm.prank(alice);
         l1StandardBridge.depositETH{ value: 1 }(300, hex"");
     }
+
+    /// @notice Prevents value transfer attacks by testing deposit amounts across full range
+    ///         including zero and maximum values that could cause overflow issues.
+    /// @param _amount Random ETH amount to test boundary conditions and edge cases.
+    function testFuzz_depositETH_variousAmounts_succeeds(uint256 _amount) external {
+        _amount = bound(_amount, 1 wei, 1000 ether);
+        vm.deal(alice, _amount + 1 ether);
+
+        _preBridgeETH({ isLegacy: true, value: _amount });
+        uint256 ethLockboxBalanceBefore = address(ethLockbox).balance;
+
+        l1StandardBridge.depositETH{ value: _amount }(50000, hex"dead");
+        assertEq(address(ethLockbox).balance, ethLockboxBalanceBefore + _amount);
+    }
+
+    /// @notice Prevents gas limit manipulation attacks that could cause bridging failures
+    ///         or enable DoS conditions by testing extreme gas limit values.
+    /// @param _gasLimit Random gas limit to test DoS and failure boundary conditions.
+    function testFuzz_depositETH_extremeGasLimits_succeeds(uint32 _gasLimit) external {
+        _gasLimit = uint32(bound(uint256(_gasLimit), 21000, 10000000));
+
+        uint256 ethLockboxBalanceBefore = address(ethLockbox).balance;
+
+        vm.expectEmit(address(l1StandardBridge));
+        emit ETHDepositInitiated(alice, alice, 1 ether, hex"dead");
+
+        vm.expectEmit(address(l1StandardBridge));
+        emit ETHBridgeInitiated(alice, alice, 1 ether, hex"dead");
+
+        vm.prank(alice, alice);
+        l1StandardBridge.depositETH{ value: 1 ether }(_gasLimit, hex"dead");
+        assertEq(address(ethLockbox).balance, ethLockboxBalanceBefore + 1 ether);
+    }
+
+    /// @notice Prevents address validation bypass attacks using edge case addresses
+    ///         that could circumvent security checks or cause unexpected behavior.
+    /// @param _recipient Random recipient address to test edge cases and validation.
+    function testFuzz_depositETH_maliciousRecipients_succeeds(address _recipient) external {
```

Comment:
Same comment as elsewhere - the function name is really confusing. It just looks like this is testing normal deposits and nothing about the recipient is malicious.